lib/oli_web/live/community_live/show.ex

```diff
@@ -77,4 +98,57 @@ defmodule OliWeb.CommunityLive.Show do
         {:noreply, assign(socket, changeset: changeset)}
     end
   end
+
+  def handle_event("add_collaborator", %{"collaborator" => %{"email" => email}}, socket) do
+    socket = clear_flash(socket)
+
+    attrs = %{
+      community_id: socket.assigns.community.id,
+      is_admin: true
+    }
+
+    case Groups.create_community_account_from_author_email(email, attrs) do
+      {:ok, _community_account} ->
+        socket = put_flash(socket, :info, "Admin successfully added.")
+        community_admins = Groups.list_community_admins(attrs.community_id)
+
+        {:noreply, assign(socket, community_admins: community_admins)}
+
+      {:error, error} ->
+        message =
+          case error do
+            :not_found ->
+              "Community admin couldn't be added. Author does not exist."
+
+            _ ->
+              Logger.error("Community admin failed to be added: #{inspect(error)}")
+
+              "Community admin couldn't be added. Author is already an admin or an unexpected error occurred (check logs)."
+          end
+
+        {:noreply, put_flash(socket, :error, message)}
+    end
+  end
+
+  def handle_event("remove_collaborator", %{"collaborator-id" => admin_id}, socket) do
+    socket = clear_flash(socket)
+
+    attrs = %{
+      community_id: socket.assigns.community.id,
+      author_id: admin_id
+    }
+
+    case Groups.delete_community_account(attrs) do
+      {:ok, _community_account} ->
+        socket = put_flash(socket, :info, "Admin successfully removed.")
+        community_admins = Groups.list_community_admins(attrs.community_id)
+
+        {:noreply, assign(socket, community_admins: community_admins)}
+
+      {:error, error} ->
+        Logger.error("Community admin failed to be removed: #{inspect(error)}")
+
+        {:noreply, put_flash(socket, :error, "Community admin couldn't be removed. Check logs.")}
```

Comment:
Nit but I think we could standardize the message when saying `"Check logs"`, either this way or between parentheses as above.
Actually, I'm not sure if suggesting to see the logs is a good idea, since that's not something authors could do. Thoughts?

Reply:
We are not logging in other places, so maybe we can remove it, what do you think? If not we can be consistent with the message and only show it for admins